### PR TITLE
[VsCoq1] Webpack Client and Server

### PR DESCRIPTION
**Pull request by 4ever2**
This PR rebases and slightly improves the work of @fakusb in #266.
Webpacking the extension resulted in ~80% decrease in loading time and ~96% decrease in size (from 8.9MB to 289KB).

This supersedes #69, #176 and #266.

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -8,20 +8,20 @@
...
 			"runtimeExecutable": "${execPath}",
 			"args": ["--extensionDevelopmentPath=${workspaceRoot}"],
-			"outFiles": ["${workspaceRoot}/out/client/**/*.js"],
-			"preLaunchTask": {
-				"type": "npm",
-				"script": "compile"
-			}
+			"outFiles": ["${workspaceRoot}/client/out/**/*.js"],
+			"sourceMaps": true,
+			"preLaunchTask": "npm: watch" //The new vscode window still needs to be reloaded after changes (Ctrl+Shift+F5 in the debugger)
```
Comment (maximedenes) [inline .vscode/launch.json:13]: Are comments allowed here?

**Comment by maximedenes** [comment]
Looks very good to me! @thery any opinion?

**Comment by thery** [comment]
@maximedenes  Yes pretty pretty good. :smile:  I'll test it and I'll merge it.

File: .vscode/launch.json (hunk 1 above)
Comment (4ever2) [reply .vscode/launch.json:13]: Yes, comments are allowed in all VS Code configuration files.

**Comment by thery** [comment]
Is this normal?
````
WARNING in ./node_modules/vscode-languageserver/lib/files.js 84:36-54
Critical dependency: the request of a dependency is an expression
 @ ./node_modules/vscode-languageserver/lib/main.js 19:11-29
 @ ./server/src/server.ts 13:32-64

WARNING in ./node_modules/vscode-languageserver/lib/files.js 319:15-28
Critical dependency: the request of a dependency is an expression
 @ ./node_modules/vscode-languageserver/lib/main.js 19:11-29
 @ ./server/src/server.ts 13:32-64
````

**Comment by 4ever2** [comment]
> Is this normal?

I don't think it is something we should worry about. It can be solved by upgrading vscode-languageserver to a more recent version.

**Comment by thery** [comment]
@4ever2  Thanks

**Comment by fakusb** [comment]
@4ever2 thanks for tying op this loose end :)

**Pull request merged**